search-service/src/test/kotlin/com/egm/stellio/search/service/TemporalEntityAttributeServiceTests.kt

```diff
@@ -735,4 +737,51 @@ class TemporalEntityAttributeServiceTests : WithTimescaleContainer {
             .expectComplete()
             .verify()
     }
+
+    @Test
+    fun `it should return UUID if entity and attribute exist`() {
+        val rawEntity = loadSampleData()
+
+        every { attributeInstanceService.create(any()) } answers { Mono.just(1) }
+        every { entityPayloadService.createEntityPayload(any(), any()) } answers { Mono.just(1) }
+
+        temporalEntityAttributeService.createEntityTemporalReferences(rawEntity, listOf(APIC_COMPOUND_CONTEXT)).block()
+
+        runBlocking {
+            temporalEntityAttributeService.checkEntityAndAttributeExistence(beehiveTestCId, INCOMING_PROPERTY).fold(
+                { fail("The referred resource has been found") },
+                { }
+            )
+        }
+    }
+
+    @Test
+    fun `it should return attribute not found if entity exist`() {
```

Comment:
maybe "it should return a left attribute not found if entity exists but not the attribute"?